rest/data/version.go

```diff
@@ -54,6 +54,24 @@ func (vc *DBVersionConnector) FindVersionById(versionId string) (*version.Versio
 	return v, nil
 }
 
+func (vc *DBVersionConnector) FindActivatedVersionsByProjectId(
+	projectId string,
```

Comment:
nit: can you keep the parameters on one line?

Reply:
Sure, but the line will almost exceed line length limitation (according to any codestyle).